README.md

```diff
@@ -21,10 +21,14 @@ We're going to build it, the open source way, of course! This repo is under MIT
 
 ## Job postings:
 Have you come across any job posting at an open source company? Feel free to add it below:
-- Note: Keep all the job posts fresh, with a limit of 3 months range
-- The first job is an example of how to fill it out
+- > Note: Keep all the job posts fresh, with a limit of 3 months range
+- > First Job is an example of how to fill it out
 
 | Position Name       | Company           | Tech Stack       | Expiring Date    | Link to apply             |
 | :-------------:     | :-------------:   | :--------:       | :--------------: | :----------:              |
-| Frontend Developer  | OpenZeppelin      | React, Solidity   |  December 1, 2022   | [Apply](https://www.openzeppelin.com/jobs/opening?gh_jid=5283107003)|
-|                     |                   |                  |                  |                           |
+| Frontend Developer  | OpenZeppelin      | React, Solidity   |  December, 2022   | [Apply](https://www.openzeppelin.com/jobs/opening?gh_jid=5283107003)|
+| Senior Full-Stack Engineer |  Amplication   |  MERN, AWS, Google Cloud | January 1, 2023 | [Apply](https://amplication.breezy.hr/p/77313a7acfa1-senior-full-stack-engineer) |
```

Comment:
they don't specifically mention MERN.

also, they don't say AWS AND Google Cloud.

could you remove the other things you changed?

and limit the duration to 3 months?